client/components/footer/Data.js

```diff
@@ -0,0 +1,57 @@
+import EmailIcon from "@mui/icons-material/Email";
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import styles from "../../styles/Footer.module.css"
+
+
+const Data=[
+    {
+        id : 1,
+        title: "Quick Access",
+        line1: "B.Tech Course",
```

Comment:
Why not make this an array?
```suggestion
        "quick_access": [
            {
                "line": "B.Tech Course",
                "link": ""
            },
            // .. and so on
        ],
```